common/models/role.js

```diff
@@ -240,6 +241,15 @@ module.exports = function(Role) {
     Role.isAuthenticated(context, callback);
   });
 
+  Role.registerResolver(Role.EVERYONE_PROMISE, function(role, context) {
```

Comment:
I find this suspicious. Where is `EVERYONE_PROMISE` coming from? I would not expect that adding promise support would require a new role. Am I missing something?